fds/services/git_service.py

```diff
@@ -55,4 +55,25 @@ def add(self, add_argument: str) -> Any:
             execute_command(["git", "add", ignore_file])
 
     def commit(self, message: str) -> Any:
+        """
+        Commit to Git
+        :param message: Commit message
+        :return:
+        """
```

Comment:
Not critical, but I hate code comments like this. Prefer to have just code.
![image](https://user-images.githubusercontent.com/611655/119823844-eef57880-befd-11eb-9871-27dd306a9dc4.png)

Reply:
Should I delete all the comments on this page then? They all have the same self evident style

Reply:
If they're all like this then yes

Reply:
Done